client/src/cmdhficlass.c

```diff
@@ -4498,7 +4498,7 @@ static command_t CommandTable[] = {
     {"-----------", CmdHelp,                    AlwaysAvailable, "---------------------- " _CYAN_("utils") " ----------------------"},
     {"configcard",  CmdHFiClassConfigCard,      IfPm3Iclass,     "Reader configuration card"},
     {"calcnewkey",  CmdHFiClassCalcNewKey,      AlwaysAvailable, "Calc diversified keys (blocks 3 & 4) to write new keys"},
-    {"encode",      CmdHFiClassEncode,          AlwaysAvailable, "Encode binary wiegand to block 7"},
+    {"encode",      CmdHFiClassEncode,          IfPm3Iclass,     "Encode binary wiegand to block 7"},
```

Comment:
users wanted to see the encoded blocks only,  hence it is available in offline

Reply:
I think you're confusing this with "hf iclass encrypt" ?
"hf iclass encode" requires the proxmark to be plugged in, doesn't show anything otherwise.
<img width="499" alt="image" src="https://github.com/RfidResearchGroup/proxmark3/assets/8178359/47a90522-5f42-45a2-a11c-4e56e65cc2fe">

Reply:
I've also tried running this offline with the -v option but same outcome

Reply:
... I dunno,  I have no issue running it with `-v` 

![image](https://github.com/RfidResearchGroup/proxmark3/assets/8577004/d8ca526b-cdd4-4dda-91ed-6ec6f31e5de0)

Reply:
Unsure why I got an inconsistent behavior, will revert change